cfgov/jinja2/v1/_includes/atoms/hyperlink.html

```diff
@@ -1 +1,30 @@
-<a href="{{ value.url }}">{{ value.text }}</a>
+{# ==========================================================================
+
+   hyperlink
+
+   ==========================================================================
+
+   Description:
+
+   Create a hyperlink atom when given:
+
+   value:         An object with the following options for value.
```

Comment:
@anselmbradford conventions for wagtail specifically are all the page values are stored in the template variable `value`